doc/classes/TabContainer.xml

```diff
@@ -168,9 +184,15 @@
 		<member name="drag_to_rearrange_enabled" type="bool" setter="set_drag_to_rearrange_enabled" getter="get_drag_to_rearrange_enabled" default="false">
 			If [code]true[/code], tabs can be rearranged with mouse drag.
 		</member>
+		<member name="input_rollover" type="bool" setter="set_input_rollover" getter="get_input_rollover" default="false">
+			If [code]true[/code], pressing [code]ui_right[/code] on the last tab will select the first tab and pressing [code]ui_left[/code] on the first tab will select the last one.
+		</member>
 		<member name="tab_alignment" type="int" setter="set_tab_alignment" getter="get_tab_alignment" enum="TabBar.AlignmentMode" default="0">
 			Sets the position at which tabs will be placed. See [enum TabBar.AlignmentMode] for details.
 		</member>
+		<member name="tab_bar_focus_mode" type="int" setter="set_tab_bar_focus_mode" getter="get_tab_bar_focus_mode" enum="Control.FocusMode" default="2">
```

Comment:
Should probably be prefixed with just `tab_` and not `tab_bar_` to follow `tab_alignment`.